command/server/listener.go

```diff
@@ -72,7 +72,7 @@ func listenerWrapProxy(ln net.Listener, config map[string]interface{}) (net.List
 	return newLn, nil
 }
 
-func listenerWrapTLS(
+func ListenerWrapTLS(
```

Comment:
Same comment basically -- I think we should move the listener stuff out somewhere common.